project.clj

```diff
@@ -20,7 +20,8 @@
   :profiles
   {:provided
    {:dependencies
-    [[cd.go.plugin/go-plugin-api "19.7.0"]
+    [[amperity/vault-clj "0.7.0"]
```

Comment:
This must be a real dependency, or it won't be included in the plugin JAR that gets built. "Provided" dependencies are ones you build against, but don't ship (presumably because the execution environment already has a copy).

Reply:
Oh thanks! :)